packages/server/test/support/helpers/e2e.js

```diff
@@ -286,6 +286,7 @@ const localItFn = function (title, opts = {}) {
   opts.browser = normalizeToArray(opts.browser)
 
   const DEFAULT_OPTIONS = {
+    exit: process.env.EXIT != null ? process.env.EXIT : null,
```

Comment:
when will `process.env.EXIT` be null? AFAIK, members of `process.env` can only ever be `undefined` or a `string`

Reply:
well `!= null` is just how I check for `undefined` :^) 

its what coffeescript converts existent checks to

Reply:
![image](https://user-images.githubusercontent.com/1151760/75800147-14212880-5d47-11ea-9044-6e146a32caed.png)

Reply:
It's a loose check that means "not `undefined` or `null`". If anything, shouldn't we be coercing it into a boolean?

Reply:
fine